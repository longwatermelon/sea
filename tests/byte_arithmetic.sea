fn main() -> int {
    let a:byte = 20b;
    let b:byte = 3b;

    let sum:byte = a + b;
    let diff:byte = a - b;
    let prod:byte = a * b;

    return sum + diff + prod;
}
