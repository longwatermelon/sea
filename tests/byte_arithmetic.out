RETURN_CODE:100
STDOUT:
